ibmcloud/README.md

```diff
@@ -360,23 +360,7 @@ $ terraform apply
 >    ```
 
 ### Enabling Attestation agent and Authenticated Registry
-> **Prerequisites:**
```

Comment:
Please could we re-add the pre-requisites to the file, as we have to be ssh'd into the worker to get the authenticated changes to work.

Reply:
please TAL